inbc-program/inbc/command/config_command.py

```diff
@@ -17,7 +17,7 @@
 from inbm_common_lib.utility import get_canonical_representation_of_path
 from inbm_vision_lib.constants import CACHE_MANAGEABILITY, CONFIG_GET, CONFIG_SET
 from inbm_vision_lib.request_message_constants import CONFIGURATION_SUCCESSFUL_MESSAGE_LIST, \
-    CONFIGURATION_FAILURE_MESSAGE_LIST, CONFIGURATION_NO_NODES_FAILURE
+    CONFIGURATION_FAILURE_MESSAGE_LIST, FOTA_NO_NODES_FAILURE
```

Comment:
Let's name this just 'NO_NODES_FAILURE' since this is in the configuration section and the other is in the FOTA section.  A more generic name.